src/components/__snapshots__/Footer.test.jsx.snap

```diff
@@ -934,6 +956,17 @@ exports[`<Footer /> renders correctly renders without a language selector in es
             Accessibility Policy
           </a>
         </li>
+        <li>
+          <button
+            className="ot-sdk-show-settings hide-one-trust btn btn-primary"
+            disabled={false}
+            id="ot-sdk-btn"
```

Comment:
@zzzarius This all looks good to me, but I would remove the OneTrust specific naming (e.g. `ot-*`) in case the OneTrust JS was binding to it.